docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -96,22 +97,27 @@ The default `logs` policy is designed to prevent the creation of many tiny daily
 indices. You can modify the policy to meet your performance requirements and
 manage resource usage.
 
-. Activate the warm phase and click **Settings**.
+. Activate the warm phase and click **Advanced settings**.
 +
 --
-.. Set **Move data into phase when** to **30 days**. This moves indices to the
-warm tier 30 days after rollover.
+.. Set **Move data into phase when** to **30 days old**. This moves indices to
+the warm tier 30 days after rollover.
 
-.. Enable **Replicas** and change **Number of replicas** to **1**.
+.. Enable **Set replicas** and change **Number of replicas** to **1**.
 
-.. Enable **Force merge** and set **Number of segments** to **1**.
+.. Enable **Force merge data** and set **Number of segments** to **1**.
 
 [role="screenshot"]
 image::images/ilm/tutorial-ilm-modify-default-warm-phase-rollover.png[Add a warm phase with custom settings]
 --
 
-. Activate the delete phase and set **Move data into phase when** to **90**
-days. This deletes indices 90 days after rollover.
+. In the warm phase, click the trash icon to enable the delete phase.
++
+[role="screenshot"]
+image::images/ilm/tutorial-ilm-enable-delete-phase.png[Enable the delete phase]
```

Comment:
I just created an ES instance based on the most recent 8.0.0-SNAPSHOT, and the header text is **Delete Data**. However, it seems that Jean-Louis' changes are [present in `master`](https://github.com/elastic/elasticsearch/blob/master/docs/reference/images/ilm/tutorial-ilm-delete-rollover.png) and should be propagated to the next SNAPSHOT.

![image](https://user-images.githubusercontent.com/25848033/108540845-b1a29e80-72af-11eb-8667-b69972908af7.png)